### DISCUSSION TITLE
Filters is running slow

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#13312] [General]
Hi,
I got an alert :  (0.14.0  (stable))
![immagine](https://github.com/user-attachments/assets/74910bc7-c423-4004-8b7c-8773d9708baf)
How to fix it?  

No issue with the 0.13 version at all (same config)

Thank you!

**Comment by hawkeye217** [comment]
Please open a support request discussion and fill out all of the requested information so we can help.

**Comment by phlnil** [comment]
done and solved ! Thank you
